doc/tests.rst

```diff
@@ -725,11 +725,14 @@ default list of paths is:
 - The temporary directory for the current script will be replaced by ``$TMPDIR``
 
 To add additional paths to this sanitation mechanism, it's sufficient to modify
-the standard BUILD_PATH_PREFIX_MAP_ environment variable. For example:
+the standard BUILD_PATH_PREFIX_MAP environment variable. But one must
```

Comment:
shouldnt it be
```
`BUILD_PATH_PREFIX_MAP`
```
rather than
```
BUILD_PATH_PREFIX_MAP_
```

Reply:
No because it's a reference to a link at the end of the file:
```
.. _BUILD_PATH_PREFIX_MAP: https://reproducible-builds.org/specs/build-path-prefix-map/
```

Reply:
I'm a little confused by this. Previous code (before my change) was:

```
the standard BUILD_PATH_PREFIX_MAP_ environment variable. For example:
```

with the underscore at the end. The reference link has the underscore at the beginning. Also, in the source code there is this declaration (in dune_util/build_path_prefix_map0.ml):

```
let _BUILD_PATH_PREFIX_MAP = "BUILD_PATH_PREFIX_MAP"
```

But I did check and without my changes the link to the BUILD_PATH_PREFIX_MAP standard does work.
Maybe I don't understand the .rst conventions. Could you explain?

Reply:
I've made that change in my upcoming push.